### PR TITLE
Update translation of security page (es)

### PR DESCRIPTION
**Pull request by agarcia-caicedo**
Updated the security information in the security Spanish translation file

### CODE REVIEW

File: es/security/index.md [hunk 1]
```diff
@@ -9,15 +9,28 @@ Aquí podrás encontrar información relacionada a problemas de seguridad de Rub
...
-privada de email. Los problemas reportados solo serán publicados hasta que
-se liberen las correcciones correspondientes.
+Las vulnerabilidades de seguridad en el lenguaje de programación Ruby deben informarse a través de nuestra página del [programa de recompensas en HackerOne](https://hackerone.com/ruby). Asegúrese de leer los detalles específicos sobre el alcance de
+nuestro programa antes de informar un problema. Cualquier problema válido
+será publicado después de las correcciones.
+
+Si ha encontrado un problema que afecta a uno de nuestros sitios web, por favor
+informarlo [a través de GitHub](https://github.com/ruby/www.ruby-lang.org/issues/new).
+
+Si ha encontrado un problema que afecta a una gema de rubí específica, siga las
```
Comment (vtamara) [inline es/security/index.md]: de rubi -> Ruby

File: es/security/index.md [hunk 2]
```diff
@@ -9,15 +9,28 @@ Aquí podrás encontrar información relacionada a problemas de seguridad de Rub
...
+[instrucciones en RubyGems.org](http://guides.rubygems.org/security/#reporting-security-vulnerabilities).
+
+Si necesita ponerse en contacto con el equipo de seguridad directamente por fuera
+de HackerOne, puede enviar un correo electrónico (en inglés) a security@ruby-lang.org
+([la clave pública PGP](/ security.asc)), que es una lista de correo privada.
+
+Los miembros de la lista de correo son personas que proporcionan a Ruby. Los miembros deben ser personas individuales, no se permiten listas de correo.
```
Comment (vtamara) [inline es/security/index.md]: que proporcionan a Ruby -> que proveen Ruby (contribuyentes y autores de otras implementaciones de Ruby, distribuidores, plataformas PaaS).

File: es/security/index.md [hunk 3]
```diff
@@ -9,15 +9,26 @@ Aquí podrás encontrar información relacionada a problemas de seguridad de Rub
...
+informarlo [a través de GitHub](https://github.com/ruby/www.ruby-lang.org/issues/new).
+
+Si ha encontrado un problema que afecta a una gema de Ruby específica, siga las
+[instrucciones en RubyGems.org](http://guides.rubygems.org/security/#reporting-security-vulnerabilities).
+
+Si necesita ponerse en contacto con el equipo de seguridad directamente por fuera
+de HackerOne, puede enviar un correo electrónico (en inglés) a security@ruby-lang.org
+([la clave pública PGP](/ security.asc)), que es una lista de correo privada.
+
+Los miembros de la lista de correo son personas que que proveen Ruby (contribuyentes y autores de otras implementaciones de Ruby, distribuidores,plataformas PaaS). Los miembros deben ser personas individuales, no se permiten listas de correo.
```
Comment (vtamara) [inline es/security/index.md]: que que -> que

File: es/security/index.md [hunk 4]
```diff
@@ -9,15 +9,26 @@ Aquí podrás encontrar información relacionada a problemas de seguridad de Rub
...
 
 ## Problemas conocidos
 
 A continuación una lista de los problemas conocidos más recientes:
 
 {% include security_posts.html %}
 
-See [the English page](/en/security/) for prior security related posts.
+Ve [la página en inglés](/en/security/) para publicaciones anteriores relacionadas con la seguridad.
```
Comment (vtamara) [inline es/security/index.md]: Ve -> Vaya a

**Comment by stomar** [comment]
When amending, please fix the commit message (see e.g. new PR title). Also, wrap lines at 80 characters.

**Comment by stomar** [comment]
Also have a look at the [Wiki](https://github.com/ruby/www.ruby-lang.org/wiki) for contribution guidelines.

**Pull request merged**